src/components/ResultsItemDetails.vue

```diff
@@ -49,12 +46,19 @@ export default {
   methods: {
     savePark() {
       this.$store.commit('savePark', this.park)
+    },
+    determinePhoto() {
+      const photoRef = this.park.photos[0].photo_reference
```

Comment:
Let's remember that we'll need the shape of the saved parks data to include `park.photos[0].photo_reference` if the backend ends up removing unnecessary properties when saving parks to the database

Reply:
Great point.  Hopefully we won't have to worry about this, but it's good to keep in mind in the event that we do.

Reply:
Thank you for the update with this! I know it was a bit strange and very time consuming to get it working, glad that it is working though!